Add Estimating Cost document (#359)
hashicorp/terraform-docs-common: website/docs/cloud-docs/overview/estimating-cost.mdx

---

> page_title: Estimating Costs - Plans and Features - Terraform Cloud

Reviewer: ```suggestion
page_title: Estimate Terraform Cloud Costs - Plans and Features - Terraform Cloud
```

description: >-

>    estimate and manage your costs on Terraform Cloud 

Reviewer: ```suggestion
   Estimate and manage your costs on Terraform Cloud 
```

   using Managed Resource pricing on pay-as-you-go billing
---

# Estimating Cost

Terraform Cloud offers a flat rate per managed resource per hour for **Standard Edition** users on pay-as-you-go billing. 

## What is a Managed Resource?

> A “Managed Resource” or “Resources Under Management (RUM)” is a resource in a Terraform Cloud-managed state file where `mode = "managed"`. Terraform Cloud counts a resource as part of this count starting from the first `terraform plan` or `terraform apply` operation on the resource. Terraform Cloud does not include resources defined as a `null_resource` or `terraform_data` resource in the total managed resource count.

Reviewer: ```suggestion
A “Managed Resource” or “Resources Under Management (RUM)” is a resource in a Terraform-Cloud-managed state file where `mode = "managed"`. Terraform Cloud counts a resource as part of this count starting from the first `terraform plan` or `terraform apply` operation on the resource. Terraform Cloud does not include resources defined as a `null_resource` or `terraform_data` resource in the total managed resource count.
```

Do we create resources from a terraform plan?

Examples of managed resources include resources provisioned by a Terraform provider (e.g. an AWS VPC), by [count](/terraform/language/meta-arguments/count) and [for_each](/terraform/language/meta-arguments/for_each) meta-arguments, and resources provisioned by [modules](/terraform/language/modules) and [no-code ready modules](/terraform/cloud-docs/no-code-provisioning/module-design).  

## Standard Edition Cost on Pay-As-You-Go Billing

> Each managed resource is charged on a per-hour basis, from the time a managed resource is provisioned until it is destroyed. Each partial hour is billed as a full hour.  Thus, the peak number of managed resources in a given hour is used for billing purposes. 

Reviewer: ```suggestion
HashiCorp charges for each managed resource on a per-hour basis, from the time a managed resource is provisioned until it is destroyed. Each partial hour is billed as a full hour.  Thus, the peak number of managed resources in a given hour is used for billing purposes. 
```
